common/dynamicconfig/cmp.go

```diff
@@ -41,18 +41,6 @@ func intCompareEquals(a, b interface{}) bool {
 	return aVal == bVal
 }
 
-func boolCompareEquals(a, b interface{}) bool {
```

Comment:
actually nothing in this file is used at all. you can delete the whole thing plus test